activesupport/lib/active_support/execution_wrapper.rb

```diff
@@ -91,7 +91,7 @@ def self.wrap(source: "application.active_support")
       begin
         yield
       rescue => error
-        error_reporter.report(error, handled: false, source: source)
+        error_reporter&.report(error, handled: false, source: source)
```

Comment:
I was prototyping possible fix myself and came up with a similar solution but with an additional public method like
```ruby
    def self.report_error(error, handled:, source:)
      error_reporter.report(error, handled: false, source: source)
    end
```

So `Reloader` could override it like
```ruby
    def self.report_error(...)
      # do nothing
    end
```

But I guess that's an overkill